src/Microsoft.Azure.VirtualKubelet.Edge.Provider/EdgeProvider.cs

```diff
@@ -68,10 +68,48 @@ void PopulateModuleConfig(Corev1ConfigMapList configMapList, string moduleName,
         {
             // see if there is an entry for this module in the config maps
             Corev1ConfigMap configMap = configMapList.Items.Where(cm => cm.Metadata.Name == moduleName).FirstOrDefault();
-            module.RestartPolicy = configMap?.Data?["restartPolicy"] ?? "on-unhealthy";
-            module.Version = configMap?.Data?["version"] ?? "1.0";
-            module.Status = configMap?.Data?["status"] ?? "running";
-            module.Settings.CreateOptions = configMap?.Data?["createOptions"] ?? "{}";
+            if (configMap != null && configMap.Data !=null)
```

Comment:
Super nit: white space around binary operators.
```csharp
if (configMap != null && configMap.Data != null)
                                          ^
```

Reply:
fixed the missing white space